priv/templates/phx.gen.html/index.html.eex

```diff
@@ -3,15 +3,15 @@
 <table>
   <thead>
     <tr>
-<%= for {k, _} <- schema.attrs do %>      <th><%= Phoenix.Naming.humanize(Atom.to_string(k)) %></th>
+<%= for {k, _} <- schema.attrs, k not in schema.redacts do %>      <th><%= Phoenix.Naming.humanize(Atom.to_string(k)) %></th>
```

Comment:
Similar to the GDPR above, I wouldn't necessarily remove redacts from here. I think it is fine to keep them here, it will be clearer in the development experience and they can always remove it later.

Reply:
That makes sense. Removing it manually after generation is easy, but manually adding it if we don't generate it is a pain. I will remove this change. I will do the same in the show template.